java/src/main/native/ai_onnxruntime_OnnxTensor.h

```diff
@@ -0,0 +1,133 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```

Comment:
If it's generated, can it be generated on the fly?

Reply:
These files were originally generated in https://github.com/microsoft/onnxruntime/pull/2866/files#diff-81d4ae4b5ee78c2401305edd9d2bcde6L44

However, I took the java stuff out of cmake. The gradle compileJava task generates these files now: https://github.com/microsoft/onnxruntime/pull/2866/files#diff-9d225bc39d80f8e6874619bca156dfc7R39

I attempted to have CMake call gradlew compileJava to generate the headers then compile the onnxruntime4_jni target and then run the gradle cmakeBuild task upon after that is completed, but I had trouble getting onnxruntime4_jni to depend on the gradlew compileJava task. I'll take a look at trying this again shortly.

Reply:
This is why the build moved entirely to CMake as orchestrating the CMake & gradle dance necessary to have it emit the headers on a clean checkout was tricky. It basically always failed the first time through, such that I had to run gradle skipping tests and then rerun it.